From 05f400b55dc4c3c64f5bc089bdbbfa805922cea4 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Wed, 21 Apr 2021 15:25:40 +0530
Subject: [PATCH] Fix the wlan build error

2 new args have been added to send_and_recv_msgs function.
Passing NULL here.

Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 wpa_supplicant_8_lib/driver_cmd_nl80211.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wpa_supplicant_8_lib/driver_cmd_nl80211.c b/wpa_supplicant_8_lib/driver_cmd_nl80211.c
index 20cf8dc..d7a34fc 100644
--- a/wpa_supplicant_8_lib/driver_cmd_nl80211.c
+++ b/wpa_supplicant_8_lib/driver_cmd_nl80211.c
@@ -166,7 +166,7 @@ static int nl80211_vendor_cmd(void *priv, unsigned int vendor_id,
 		goto fail;
 	}
 
-	ret = send_and_recv_msgs(drv, msg, vendor_reply_handler, buf);
+	ret = send_and_recv_msgs(drv, msg, vendor_reply_handler, buf, NULL, NULL);
 	if (ret)
 		wpa_printf(MSG_DEBUG, "nl80211: vendor command failed err=%d",
 			   ret);
-- 
2.17.1

